seed/tests/test_column_views.py

```diff
@@ -152,6 +152,22 @@ def test_get_all_columns(self):
         # randomly check a column
         self.assertIn(expected, data)
 
+    def test_column_units_added(self):
+
+        responseWithoutUnits = self.client.get(reverse('api:v3:columns-list'), {
+            'organization_id': self.org.id,
+            'display_units': 'false'
+        })
+        columnWithoutUnits = next((x for x in json.loads(responseWithoutUnits.content)['columns'] if x['display_name'] == 'Source EUI Modeled'), None)
+        self.assertEqual(columnWithoutUnits['display_name'], 'Source EUI Modeled')
```

Comment:
If you're using `display_name` to capture the dictionary representation of a desired column object, I don't think it's advisable to make an assertion on the `display_name` value. In a sense, it's kind of like testing the `next()` Python method as well as the existence of a column with that `display_name` - that existence check is valuable though.

Maybe we can use `x['column_name'] == 'source_eui_modeled'` instead?

Reply:
Good point, thanks for catching this.